### DISCUSSION TITLE
access images uploaded into static without rebuild

### DISCUSSION THREAD

**Discussion by PlopTheReal** [sveltejs/kit#5612] [Q&A] [answered]
I'm creating a CMS that permit images upload into the default static folder but I can't access the images uploaded once the build has been made (the url pointing to the new asset fallsback to the index page).
I have to rebuild the site to make it available.
Is there is any ways to deliver whatever is available in the static path on demand?

**Comment by PlopTheReal** [comment, marked as answer]
Ok I got it sorted by using a node adapter and sirv.

**Comment by ayuancool** [comment]
Why can't we provide an out of the box 'upload' directory?

**Reply by rafalkrupinski** [reply]
These days you'd normally upload to an external storage

**Reply by ayuancool** [reply]
The ability to temporarily store files on the intranet or to do some work before uploading to external storage is still necessary
